### PR TITLE
Add ScapeGPT plugin to plugin hub

### PR DESCRIPTION
**Pull request by polyphilz**
A simple ChatGPT-esque plugin that lets users ask any OSRS-related question and receive an answer that leverages OpenAI's chat models augmented on data from all 27k+ articles on the OSRS wiki.

### CODE REVIEW

**Comment by Ben-Colwell** [comment]
You need to fix these issues

net.runelite.pluginhub.packager.PluginBuildException: scapegpt: plugin uses terminally deprecated APIs:
Do not create fresh Gson instances, always @Inject the client's Gson.
You can customize it by calling .newBuilder() on it
Do not create fresh OkHttpClients - always @Inject the client's OkHttpClient
	at net.runelite.pluginhub.packager.PluginBuildException.of(PluginBuildException.java:54)
	at net.runelite.pluginhub.packager.Plugin.assembleManifest(Plugin.java:720)
	at net.runelite.pluginhub.packager.Packager.buildPlugin(Packager.java:183)
	at net.runelite.pluginhub.packager.Packager.lambda$buildPlugins$0(Packager.java:122)
	at java.base/java.lang.Thread.run(Thread.java:829)

**Comment by rbnsl** [comment]
Ugh sorry I'm bad at dependency injection lol gimme a bit I'll figure it out

**Comment by raiyni** [comment]
Don't use https://github.com/polyphilz/scapegpt/blob/66e124be600c998c0eca78d33a309fced2cf7c8f/src/main/java/com/rohanbansal/ScapeGptPlugin.java#L51

**Comment by polyphilz** [comment]
> Don't use https://github.com/polyphilz/scapegpt/blob/66e124be600c998c0eca78d33a309fced2cf7c8f/src/main/java/com/rohanbansal/ScapeGptPlugin.java#L51

Removed sending of any RL auth specifics outright to server; don't need them, this was leftover from a previous experiment.

File: plugins/scapegpt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/polyphilz/scapegpt.git
+commit=64ed578d0370253b408634a1c03944e4425895d4
```
Comment (Nightfirecat) [inline plugins/scapegpt:2]: This needs a warning as it has a default configuration which submits requests to non-verified servers.
```suggestion
commit=64ed578d0370253b408634a1c03944e4425895d4
warning=This plugin submits your IP address to a server not controlled or verified by the RuneLite developers.
```

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

**Pull request merged**